<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateShoal</class>
 <widget class="QWidget" name="CreateShoal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>123</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>201</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="shoalTagLabel">
       <property name="text">
        <string>Shoal Tag</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="shoalTagLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="shoalDescriptionLabel">
       <property name="text">
        <string>Shoal Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="shoalDescriptionLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="shoalThemeLabel">
       <property name="text">
        <string>Shoal Theme</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="shoalThemeComboBox">
       <item>
        <property name="text">
         <string>Froth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Deep Sea</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ocean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seaweed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beach</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Riverbed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jelly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coral</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>123</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
